Dont pull this just starting to work on it

grant2258 opened the pull request.

grant2258 committed:
start working on this rf5c68 first

grant2258 committed:
return sample number playing

grant2258 committed:
debuggy

grant2258 committed:
git the data sheet mystery of the rf5c86 solved

KMFDManic commented:
@grant2258 No problem, ping me in a response comment to this message when it is alright to pull:)

KMFDManic committed:
Custom OST Testing Grounds

MAME 2003 Xtreme will take the path of the more experimental Custom OSTs, hereby now being Testing Grounds.  Credits Updated, etc.

mahoneyt944 committed:
Street fighter 1 OST

mahoneyt944 committed:
outrun OST update

mahoneyt944 committed:
SF2 and Final Fight OST update

mahoneyt944 committed:
Update ddragon.c

mahoneyt944 committed:
Update system18.c

mahoneyt944 committed:
Update midyunit.c

mahoneyt944 committed:
Update midtunit.c

mahoneyt944 committed:
Update williams_sndhrdw.c

mahoneyt944 committed:
Update williams.h

mahoneyt944 committed:
Update williams_sndhrdw.c

grant2258 committed:
fix the rf5c86 sample calc

grant2258 committed:
finish up

grant2258 commented:
Thats you good to go @KMFDManic. setting volumes will relative to the global mixer. Give jpark and sonic a try music is pleasant now.

* fixed rf5c86 some games like ga2 and slipstream need there volumes upped
* adding sample number to sample is is playing you need to subtract one from this result if using it to play a sample.
* added volumes to set via global mixer settings. 


for example:

https://github.com/KMFDManic/mame2003-xtreme/blob/0f7b6d128629b7352bb4b29f20a1698a1e40cc62/src/ost_samples.c#L509 this is the global mixer volume.

sample set volume is is the channel volume. Its now done right retaliative to the global mixer. All volumes are set to 39 that where the old mixing was set.

KMFDManic commented:
Completely awesome, @grant2258 the musics were a little irritating being not at proper levels. Cant wait to try this stuff out:) Ill get more samples together for u to have fun with.  Any specific games u might like to see custom samples for?
